cmd/minikube/cmd/tunnel.go

```diff
@@ -116,6 +122,36 @@ var tunnelCmd = &cobra.Command{
 	},
 }
 
+func cleanupLock() {
+	if lockHandle != nil {
+		err := lockHandle.Unlock()
+		if err != nil {
+			out.Styled(style.Warning, fmt.Sprintf("failed to release lock during cleanup: %s", err.Error()))
+		}
+	}
+}
+
+func mustLockOrExit() {
+	tunnelLockPath := filepath.Join(localpath.MiniPath(), ".tunnel_lock")
+
+	_, err := os.OpenFile(tunnelLockPath, os.O_RDWR|os.O_TRUNC, 0600)
+	if err != nil {
+		if os.IsNotExist(err) {
+			_, err = os.Create(tunnelLockPath)
+			if err != nil {
+				exit.Error(reason.SvcTunnelStart, fmt.Sprintf("error creating lock for tunnel file (%s)", tunnelLockPath), err)
+			}
+		} else {
+			exit.Error(reason.SvcTunnelStart, fmt.Sprintf("error creating lock for tunnel file (%s)", tunnelLockPath), err)
+		}
```

Comment:
```suggestion
		if !os.IsNotExist(err) {
			exit.Error(reason.SvcTunnelStart, fmt.Sprintf("error creating lock for tunnel file (%s)", tunnelLockPath), err)
		}
		_, err = os.Create(tunnelLockPath)
		if err != nil {
			exit.Error(reason.SvcTunnelStart, fmt.Sprintf("error creating lock for tunnel file (%s)", tunnelLockPath), err)
		}
```
Can prevent a lot of this nesting by negating the first `if`

Reply:
Thanks, fixed!